common/discordutil.js

```diff
@@ -57,7 +61,28 @@ module.exports = {
         console.log(channelID, classCode);
 
         file.ids[channelID] = classCode;
-        this.writeToFile();
+        this.writeToFile(pathToJson, file);
```

Comment:
Also in line 74 and 85, when `writeToFile` fails, it returns false. What does the bot do in this case when it receives false? Will it fail silently (like, it'll just show the bot thinking endlessly?)